tests/cli/workflow/mip/test_cli_mip_store.py

```diff
@@ -20,18 +21,20 @@ def test_store_no_config(cli_runner: CliRunner, mip_store_context: dict, caplog)
 
 
 def test_store_analysis_exception(
-    cli_runner: CliRunner, mip_store_context: dict, mip_configs: dict
+    cli_runner: CliRunner, mip_store_context: dict, mip_configs: dict, caplog
 ):
     """Test that the analysis function enters the exception clause"""
-
-    # WHEN we run store on a case without deliverables
-    result = cli_runner.invoke(analysis, [str(mip_configs["purplesnail"])], obj=mip_store_context)
-    # THEN we should be informed that mandatory files are missing
-    assert "Mandatory files are missing" in result.output
-    # THEN we should not include any files in housekeeper
-    assert "included files in Housekeeper" not in result.output
-    # THEN the exit code should be EXIT_FAIL
-    assert result.exit_code == EXIT_FAIL
+    with caplog.at_level("ERROR", "INFO"):
```

Comment:
Why not 
```suggestion
caplog.set_level(logging.ERROR)
```
and avoid indenting everything?

Reply:
this allows to check outputs of both log levels